content/en/docs/concepts/configuration/configmap.md

```diff
@@ -273,5 +273,5 @@ to the deleted ConfigMap, it is recommended to recreate these pods.
 * Read [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/).
 * Read [The Twelve-Factor App](https://12factor.net/) to understand the motivation for
   separating code from configuration.
-
-
+* ConfigMap is a top-level resource in the Kubernetes REST API.
+  The {{< api-reference page="config-and-storage-resources/config-map-v1" >}} object definition describes the object in detail.
```

Comment:
why are we splitting the 'configmap' word into 'config-map'?
This will cause a lot of confusion in the future.

Reply:
This is an automtatic transformation from PascalCase to kebab-case, and the resource name is ConfigMap, transformed to config-map